static/js/click_handlers.js

```diff
@@ -228,7 +228,14 @@ exports.initialize = function () {
             animation: false,
         });
         elem.tooltip("show");
-        $(".tooltip, .tooltip-inner").css("max-width", "600px");
+        if ($(window).width() > 775) {
+            $(".tooltip, .tooltip-inner").css("max-width", "600px");
+        } else {
+            $(".tooltip, .tooltip-inner").css({
+                "margin-left": "15px",
+                "max-width": $(window).width() * 0.6,
```

Comment:
As of now ( In devices with screen sizes < 775px ) , I have kept the <code>max-width</code> property of tooltip to be 60% of the total screen width ( So that long tooltips don't cover up entire screen width ) .